var/spack/repos/builtin/packages/py-pycairo/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+
+
+class PyPycairo(PythonPackage):
```

Comment:
There is already a `py-py2cairo` package that appears to be a duplicate. I agree that the name should be `py-pycairo`. I would prefer to download directly from PyPI.

Reply:
ugh, the py-py2cairo package name is a bit misleading. Perhaps we can make a separate PR to rename it and merge to py-pycairo and then modify all the packages that depend on it? I fear there will be some major compatibility issues that will arise since py2cairo is for Python 2 and pycairo is for Python 3.